Sources/Remote Logging/Crash Logging/SentryEventJSException.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+import Sentry
+
+public struct JSException {
+    public let type: String
+    public let message: String
+    public let stacktrace: [StacktraceLine]
+    public let context: [String: Any]
+    public let tags: [String: String]
+    public let isHandled: Bool
+    public let handledBy: String
+
+    public init(type: String, message: String, stacktrace: [StacktraceLine], context: [String : Any], tags: [String : String], isHandled: Bool, handledBy: String) {
+        self.type = type
+        self.message = message
+        self.stacktrace = stacktrace
+        self.context = context
+        self.tags = tags
+        self.isHandled = isHandled
+        self.handledBy = handledBy
+    }
+    
+    public struct StacktraceLine {
+        public let filename: String?
+        public let function: String?
```

Comment:
Should this `funciton` property be non-optional to match the one declared in `GutenbergJSException`?

Reply:
Ah, good catch, I forgot to update this struct. I'll update it asap.